### DISCUSSION TITLE
Build app Docker iamge

### DISCUSSION THREAD

**Discussion by michallm** [vercel/turborepo#603] [Help] [answered]
Is it possible to create a docker image from an application e.g. express server that depends on package from packages folder?

**Comment by Myrmod** [comment, marked as answer]
Might this help you: https://github.com/vercel/turborepo/issues/215?

I am currently trying to create a build process integrated in the gitlab ci. Essentially the same thing you want: https://github.com/vercel/turborepo/discussions/609
